docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -636,6 +636,54 @@ Here is a brief description of the fields :
+
+image::docs/images/saml-packetfence-config.png[scaledwidth="100%",alt="SAML PacketFence configuration"]
+
+Where :
+
+[options="compact"]
+* *Service Provider entity ID* is the identifier of the Service Provider (PacketFence). Make sure this matches your Identity Provider configuration.
+* *Path to Service Provider key* is the path to the key that will be used by PacketFence to sign its messages to the Identity Provider. A default one is provided under the path : `/usr/local/pf/conf/ssl/server.key`
+* *Path to Service Provider cert* is the path to the certificate associated to the key above. A self-signed one is provided under the path : `/usr/local/pf/conf/ssl/server.key`
+* *Path to Identity Provider metadata* is the path to the metadata file you transfered above (should be in `/usr/local/pf/conf/idp-metadata.xml`)
+* *Path to Identity Provider cert* is the path to the certificate of the identity provider you transfered on the server above (should be in `/usr/local/pf/conf/ssl/idp.crt`).
+* *Path to Identity Provider CA cert* is the path to the CA certificate of the identity provider you transfered on the server above (should be in `/usr/local/pf/conf/ssl/ca-idp.crt`). If the certificate above is self-signed, put the same path as above in this field.
+* *Attribute of the username in the SAML response* is the attribute that contains the username in the SAML assertion returned by your Identity Provider. The default should fit at least SimpleSAMLphp.
+* *Authorization source* is the source that will be used to match the username against the rules defined in it. This allows to set the role and access duration of the user. The 'Authentication' section of this document contains explanations on how to configure an LDAP source which can then be used here.
+
+Once this is done, save the source and you will be able to download the Service Provider metadata for PacketFence using the link 'Download Service Provider metadata' on the page.
+
+Configure your identity provider according to the generated metadata to complete the Trust between PacketFence and your Identity Provider.
+
+In the case of SimpleSAMLPHP, the following configuration was used in `metadata/saml20-sp-remote.php` : 
+
+
+  $metadata['PF_ENTITY_ID'] = array(
+    'AssertionConsumerService' => 'http://PORTAL_HOSTNAME/saml/assertion',
+    'SingleLogoutService' => 'http://PORTAL_HOSTNAME/saml/logoff',
+  );
+
+NOTE: PacketFence does not support logoff on the SAML Identity Provider. You can still define the URL in the metadata but it will not be used.
+
+Passthroughs
+^^^^^^^^^^^^
+
+In order for your users to be able to access the Identity Provider login page, you will need to activate passthroughs and add the Indentity Provider domain to the allowed passthroughs.
+
+To do so, go in 'Configuration -> Trapping', then check 'Passthrough' and add the Identity Provider domain name to the 'Passhtroughs' list.
+
+Restart PacketFence to apply the new passthroughs configuration.
```

Comment:
Well if the passthroughs have just been enabled, it requires to restart iptables and pfdns

If they were already enabled only pfdns.

I put the full restart so there is no doubt the proper services will be restarted.

The section on passthroughs still states to restart PacketFence.

The above needs to be addressed anyway since we shouldn't even have to restart pfdns after we add a passthrough as it should use pfconfig to auto refresh its values.

Should I still change it to the conditional one ?